src/main/java/seedu/task/model/task/Description.java

```diff
@@ -12,6 +12,15 @@ public class Description {
     public static final String MESSAGE_CONSTRAINTS =
             "Description must contain at least one character";
     public final String value;
+    public final String defaultValue = "No Description";
```

Comment:
Would it be better to make it a class variable?

Reply:
Updated as suggested.